pytensor/tensor/rewriting/linalg.py

```diff
@@ -377,3 +382,50 @@ def local_lift_through_linalg(
                 return [block_diag(*inner_matrices)]
             else:
                 raise NotImplementedError  # pragma: no cover
+
+
+@register_canonicalize
+@register_stabilize
+@register_specialize
+@node_rewriter([SVD])
+def local_svd_uv_simplify(fgraph, node):
```

Comment:
Suggesting more specific name

```suggestion
def svd_uv_merge(fgraph, node):
```